src/core/qfieldcloudprojectsmodel.cpp

```diff
@@ -1373,7 +1373,7 @@ void QFieldCloudProjectsModel::downloadFileConnections( const QString &projectId
     {
       hasError = true;
       errorMessageDetail = file.errorString();
-      errorMessageTemplate = QStringLiteral( "Failed to write downloaded file stored at \"%1\", fs reason" ).arg( fileName );
+      errorMessageTemplate = tr( "Failed to write downloaded file stored at \"%1\", fs reason" ).arg( fileName );
```

Comment:
```suggestion
      errorMessageTemplate = tr( "Failed to write downloaded file stored at \"%1\", fs reason:" ).arg( fileName );
```
To be aligned with the message above